x/validator/client/cli/tx_propose_disable_validator.go

```diff
@@ -0,0 +1,67 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/utils/cli"
+	"github.com/zigbee-alliance/distributed-compliance-ledger/x/validator/types"
+)
+
+var _ = strconv.Itoa(0)
+
+func CmdProposeDisableValidator() *cobra.Command {
+	var (
+		address string
+		info    string
+	)
+
+	cmd := &cobra.Command{
+		Use:   "propose-disable-validator --address [address] --info [info] ",
+		Short: "Proposes disabling of the Validator node by a Trustee.",
+		Args:  cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			fromAddr := clientCtx.GetFromAddress()
+
+			addr, err := sdk.ValAddressFromBech32(address)
+			if err != nil {
+				owner, err2 := sdk.AccAddressFromBech32(address)
```

Comment:
We should not use such ambiguous logic in tx commands. We can always get Bech32 `cosmosvaloper` address from `validator node` query result (which in turn can be executed using the address in any format) and then use the gotten value in a tx command.